docs/content/community/presentations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+type: docs
+title: "Give a Radius presentation"
+linkTitle: "Presentations"
+description: "Learn how to give a Radius presentation, with downloadable slides and assets"
+weight: 400
+---
+
+We welcome community members giving presentations on Radius and spreading the word about all the awesome Radius features!
+
+## Giving a Radius presentation
+
+1. Begin by downloading the [Radius Presentation Deck](https://github.com/radius-project/docs/raw/{{< param github_branch >}}/presentation-pack/Radius.potx). This contains slides needed to give a Radius presentation.
+1. Refer to the Radius [presentation pack directory](https://github.com/radius-project/docs/tree/{{< param github_branch >}}/presentation-pack) for any diagrams or images you need to create.
+1. Next, review the docs to make sure you understand the [concepts]({{< ref concepts >}}).
+1. Use the Radius [getting started guide]({{< ref getting-started >}}) and [samples repo](https://github.com/radius-project/samples) to show demos of how to use Radius.
+
+## Slides
+
+We offer a template PowerPoint file to get started:
+
+{{< button text="Download the Radius Presentation Deck" link="https://github.com/radius-project/docs/raw/{{< param github_branch >}}/presentation-pack/Radius.potx" newtab="true" >}}
```

Comment:
The button throws a 404 for me where as I am able to download the slides directly from the hyperlink above

Reply:
Ah yeah the param didn't render correctly. Will fix. Thanks for catching!